tools/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: base
-featured-tool: Open Gazettes
+featured-tool: opengazettes
```

Comment:
hmmm, this may not work since your tool names are now "Open Gazettes" and similar?